### ISSUE TITLE
Model failing to load with constant value tensor

### ISSUE THREAD

**Issue by fangzhou-xie** [Dobiasd/frugally-deep#447]
While trying to the load a simple model, I have encountered `Error: No keras_history on node connection. Constant-value tensors are not supported.` I have also found issue #442 that seems to be similar, but as I was using version v0.18.2, I still have the error.

I am attaching the my `.keras` file after compression here [test_trans.zip](https://github.com/user-attachments/files/20432206/test_trans.zip).

For context, I was training a small transformers model in R with `keras3` package. As part of the model, I did have layer normalization, batch normalization layers. I don't think those details matter, as they eventually will depend on the python Keras library with Tensorflow backend, and the layers I am using are all supported by the `frugally-deep` library.

Thank you very much and I'd appreciate any insights.

**Comment by Dobiasd** [comment]
Hi, and thanks for reporting!

Yes, the change with `v0.18.2` is just to have support for `invert=True` in Normalization layers. There is nothing yet implemented for constant tensors.

Could you share a minimal Python code that shows how you are using the constant tensor value?

**Comment by fangzhou-xie** [comment]
Thank you very much for your reply! While trying to create a MWE, I realized that the issue was caused by my not setting up the Embedding layer properly. It should have been a trainable layer with inputs, rather than the accidental "constant tensor layer."

Thank you again for your help and sorry for the confusion. I will close this issue now as this is resolved.

**Comment by Dobiasd** [comment]
No problem, and thanks. I'm glad you figured out the cause. 👍

Let me know when you run into any additional problems with frugally-deep.
